### ISSUE TITLE
Berryboot rpi2 openelec

### ISSUE THREAD

**Issue by danilp** [maxnet/berryboot#255]
Hi, when i install openelec 6.0 from berryboot all seems to work ok, but i start openelec, kodi isegard screen appears but next i only see bubbles wallpaper...nothing more..
Tried several times..
Any ideas?
Thx

**Comment by maxnet** [comment]
Does work fine here.

Do you have a rainbow icon to the top right of the screen? (If so, that could indicate power supply problems)

**Comment by grzechj** [comment]
I have exactly the same problem. RPI2 downloaded berryboot-20160209-pi2.zip.

**Comment by maxnet** [comment]
Does it work better if you remove max_usb_current=1 from config.txt?
Did it work with any previous version?

**Comment by danilp** [comment]
Now works...i don't know if maybe a problem with my rf keyboard...maybe???
Without the rf receiver out works ok...now with it plugged in works too...

**Comment by grzechj** [comment]
Removing max_usb_current=1 didn't resolve the issue. I haven't tried any previous versions. Also the trick with disconnecting keyboard wasn't helpful in my case. Any suggestions?

**Comment by maxnet** [comment]
> Now works...i don't know if maybe a problem with my rf keyboard...maybe???
> Without the rf receiver out works ok...now with it plugged in works too...

All wireless things do consume a lot of power.
However if that was the problem I do would expect you to receive a rainbow icon in top right corner (which is a low voltage warning).

**Comment by grzechj** [comment]
I my case the same hardware configuration works without problems when OpenElec is installed as standalone system without berryboot. Maybe there is problem with OpenElec 6.0.1 berryboot image.

**Comment by maxnet** [comment]
> Maybe there is problem with OpenElec 6.0.1 berryboot image. 

Doubt that, since we copy the OpenELEC image as-is.
OpenELEC's SYSTEM file is already in the same squashfs format as Berryboot uses internally for all images, so we do not modify it.

We do are using our own firmware and kernel version with it, and that differs by Berryboot version.
You might want to try an older Berryboot version, they are available at: https://sourceforge.net/projects/berryboot/files/?source=navbar

**Comment by grzechj** [comment]
I finally managed to make it working. I had to turn off TV before first run of OpenElec. Maybe there is a problem with HDMI CEC support for my TV model. After it loaded correctly for the first time then it works ok for all the subsequent reboots. Thanks for your patience.

**Comment by psykro** [comment]
Same issue here,

Will try as you said.

So you installed openelec, then switch off TV and reboot ?

**Comment by grzechj** [comment]
Right,  issue openelec boot from berryboot and turn off TV. You will have to turn it off more than once while HDMI will be trying to wake it up couple times during openelec and kodi startup. Optionally you can issue openelec boot from berryboot  and pull off HDMI plug.

**Comment by psykro** [comment]
Nice catch, worked like a charm.

Rebooted rpi, switched off TV, once then let it boot. When i came back TV was on and openelec booted !

WP
